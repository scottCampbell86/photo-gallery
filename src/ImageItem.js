import React, { Component } from 'react'


export default class ImageItem extends Component {
    render() {
        return (
            <div className="image-container">
                <li>
                    <h2>{this.props.image.title}</h2>
                    <p>{this.props.image.keyword}</p>
            
                    <img
                        alt={this.props.image.url}
                        src={this.props.image.url} />
                </li>
            </div>

            
        )    
    }
}