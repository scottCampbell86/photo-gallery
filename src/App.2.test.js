import React from 'react';
import ImageItem from './ImageItem.js';
import renderer from 'react-test-renderer';


it('renders correctly', () => {
  const tree = renderer
    .create(<ImageItem image={[]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
