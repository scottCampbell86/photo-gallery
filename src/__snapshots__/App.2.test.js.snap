// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="image-container"
>
  <li>
    <h2 />
    <p />
    <img />
  </li>
</div>
`;
